internal/collector/event.go

```diff
@@ -115,3 +151,9 @@ func updatedEventNb(oldEv, newEv *v1.Event) int32 {
 
 	return newEv.Count - oldEv.Count
 }
+
+func filterInvolvedObjectNs(list *metav1.ListOptions, ns string) {
+	if ns != "" {
```

Comment:
What if it is empty string? Something seems to have gone wrong then we want the user to know maybe with info log or debug log?

Reply:
Empty string is the underlying value of `metav1.NamespaceAll`. I should definitely make it clearer in the condition.